packages/ui/hooks/useUSDPrices.ts

```diff
@@ -3,59 +3,34 @@ import { useQuery } from '@tanstack/react-query';
 import axios from 'axios';
 import { SupportedChains } from 'types/dist/cjs';
 
-const ChainIdCoingeckoIdMapping = Object.entries(ChainConfigs)
-  .map(([, config]): [string, string] => [config.chainId.toString(), config.specificParams.cgId])
-  .reduce<Record<string, string>>((acc, cur) => {
-    acc[cur[0]] = cur[1];
-    acc[cur[1]] = cur[0];
-    return acc;
-  }, {});
-
-const ChainIdNativeSymbolMapping = Object.entries(ChainConfigs)
-  .map(([, config]): [string, string] => [
-    config.chainId.toString(),
-    config.specificParams.metadata.nativeCurrency.symbol,
-  ])
-  .reduce<Record<string, string>>((acc, cur) => {
-    acc[cur[0]] = cur[1];
-    acc[cur[1]] = cur[0];
-    return acc;
-  }, {});
-
-interface Price {
-  value: number;
-  symbol: string;
-}
-
 // TODO Make currency agnostic
-async function getUSDPriceOf(chainIds: SupportedChains[]): Promise<Record<string, Price>> {
-  const cgIds = chainIds.map((id) => ChainIdCoingeckoIdMapping[id]);
-
-  const { data } = await axios.get(
-    `https://api.coingecko.com/api/v3/simple/price?vs_currencies=usd&ids=${cgIds.join(',')}`
-  );
-
-  return cgIds
-    .map((cgId): [string, Price] => {
-      if (data[cgId])
-        return [ChainIdCoingeckoIdMapping[cgId], { value: data[cgId].usd, symbol: '$' }];
```

Comment:
The beauty of this `value` & `symbol` pair was to be able to show make it any currency and not hardcode it to be `usd` everyhwere.
This way we could fall back to just show stuff in `native` value as well if no `usd` price is available. 🤷